MSAL/src/native_auth/controllers/reset_password/MSALNativeAuthResetPasswordController.swift

```diff
@@ -520,7 +520,7 @@ final class MSALNativeAuthResetPasswordController: MSALNativeAuthBaseController,
             case .failed:
                 let error = PasswordRequiredError(type: .generalError)
                 self.stopTelemetryEvent(event, context: context, error: error)
-                MSALLogger.log(level: .error, context: context, format: "password poll success returned status 'failed'")
+                MSALLogger.log(level: .error, context: context, format: "Password poll success returned status 'failed'")
```

Comment:
I think this message should be "Password poll completion returned..." not "Password poll success returned..." Sorry I'd say that was my mistake originally :)

Reply:
Indeed, I didn't even process that part as I was focusing on the capitalisation one. Updated.